.snyk

```diff
@@ -0,0 +1,12 @@
+# eslint-disable prettier/prettier
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+# See: https://docs.snyk.io/snyk-cli/test-for-vulnerabilities/the-.snyk-file
+version: v1.1061.0
```

Comment:
This is supposed to be the current version of the [`snyk-policy`](https://www.npmjs.com/package/snyk-policy) package.

```suggestion
version: 1.25.0
```